async_1.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+const async_1 = exports = module.exports;
+const fs = require('fs');
+const files = ['one.txt', 'two.txt', 'three.txt'];
+const EventEmitter = require('events').EventEmitter;
+const ee = new EventEmitter();
+var testOrder = [];
+
+fs.readFile('./one.txt', (err, data) => {
+  console.log(data.toString('hex', 0, 8))
+  testOrder.push(1)
+  ee.emit('two');
+})
+
+ee.on('two', () => {
+  fs.readFile('./two.txt', (err, data) => {
+    console.log(data.toString('hex', 0, 8))
+    testOrder.push(2)
```

Comment:
You'll probably want to be collecting data into an array as well if you want to test against it.